Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -305,6 +334,41 @@ def runSelected(opt):
                       default=None,
                       action='store')
 
+    parser.add_option('--RequiresGPU',
+                      help='if GPU is reuired or not: forbidden (default, CPU-only), optional, required. For relvals, the GPU option will be turned off for optional.',
```

Comment:
For relvals, I assume that we always control what relvals should run on, w/ or w/o (specific) GPU. So optional may not fit with the relvals. But this can be changed, no solid reason here.